docs/Makefile

```
@@ -20,8 +20,8 @@ help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  html       to make standalone HTML files"
 	@echo "  htmlview   to open the index page built by the html target in your browser"
+	@echo "  htmllive   to rebuild and reload HTML files in your browser"
```

```suggestion
	@echo "  htmllive   to rebuild and reload HTML files in your browser"
	@echo "  htmlview   to open the index page built by the html target in your browser"
```
If you, and you may not, make the other suggested change to be alphabetical, maybe this group of html* targets should be in the same order?

---

Let's keep this as is -- it's going from simpler action to more complex.

And actually, I'll revert the change to the top-level `Makefile` too.